serializations.md

```diff
@@ -482,9 +504,8 @@ The channel offchain transaction is not included directly in the transaction tre
 
 #### Channel
 ```
-[ <id>               :: binary()
```

Comment:
Update required [here](https://github.com/aeternity/protocol/blob/6614f009a3d033e3f8aec6a381d16def766f7eaa/channels/ON-CHAIN.md#channel-state-tree) - and/or link to here.

Reply:
Again, just the serialization. Not the otherwise visible format.